bravado/testing/integration_test.py

```diff
@@ -473,6 +492,16 @@ def test_msgpack_support(self, swagger_http_server):
         assert response.headers['Content-Type'] == APP_MSGPACK
         assert unpackb(response.raw_bytes, encoding='utf-8') == API_RESPONSE
 
+    def test_redirects_are_not_followed(self, swagger_http_server):
```

Comment:
I would expect a test that ensures that redirects are followed as well.

Reply:
Done, but only for the `RequestsClient` and it doesn;t look like the fido client supports the ability to follow redirects.

Reply:
I think that we should be doing a small effort here to ensure that the two clients behave the same here.
I'll try to look for pointers to help you out here

Reply:
That was the conclusion I came to. The logigcal place to add support for following redirects in Fido, would be in [prepare_request_for_twisted](https://github.com/Yelp/bravado/blob/master/bravado/fido_client.py). The retuened value from that in turn goes into [fido.fetch](https://github.com/Yelp/fido/blob/master/fido/fido.py) which attempts to look like the interface for requests, but isn't fully implemented (as demonstrated by the lack of allow_redirects parameter. For the record, I have no clue how the call to fido.fetch works in fido_client.py as it seems to be missing an import.

In summary, the effort required to make fido work with redirects is much more significant that getting requests to disable redirects by default. Given the bug I initially found where following redirects causes bravado to complain about unexpected HTTP statuses, I'd say that support for following redirects at all should probably be removed in the future or it should handle redirects at a higher level, e.g. response returns redirect, so now you should expect the response(s) from the target of the redirect. As not all redirects have a single target according to the spec, that gets even more complicated to handle at a library level, so I'd argue that's an implementation decision for the user of bravado to deal with.